README.rst

```diff
@@ -181,13 +181,17 @@ In the examples above, use ``-e`` to evaluate the code in the host kernel as wel
 
 Configuration
 -------------
-``Metakernel`` subclasses can be configured by the user by adding a ``metakernel_config.py`` file to their
-``jupyter`` config path.  The base ``MetaKernel`` class offers ``plot_settings`` as a configurable trait.
+``Metakernel`` implementations can subclasses can be configured by the user.  The
```

Comment:
Looking good @blink1073! Noticed a couple typos here, should probably change

`implementations can subclasses can be configured by the user` -> `subclasses can be configured by the user`
`Subclasses can defined` -> `Subclasses can define`

Reply:
Cheers, fixed.